jarvis_files/jarvis_cmd/src/jarvis/build.py

```diff
@@ -208,38 +208,38 @@ def genTerminalLaunchCommand(script_address, ssh):
         return command + "; $SHELL'"
 
 # Functions that build and execute auton subteam code
-def launch_percep(ctx, opts, ssh):
+def launch_perception(ctx, opts, ssh):
     percep = 'jetson/percep'
-    l = 'True'
+    use_linter = 'True'
     workspace_relative_address = "jarvis_files/jarvis_cmd/launchScripts/percep"
     
-    build_dir(ctx, percep, l, opts)
+    build_dir(ctx, percep, use_linter, opts)
 
     ctx.run( genTerminalLaunchCommand(workspace_relative_address, ssh) )
     
     wait_for_click()
     return
 
-def launch_nav(ctx, opts, ssh):
+def launch_navigation(ctx, opts, ssh):
     nav = 'jetson/nav'
-    l = 'True'
+    use_linter = 'True'
     workspace_relative_address = "jarvis_files/jarvis_cmd/launchScripts/nav"
     
-    build_dir(ctx, nav, l, opts)
+    build_dir(ctx, nav, use_linter, opts)
     
     ctx.run( genTerminalLaunchCommand(workspace_relative_address, ssh) )
 
     wait_for_click()
     return
 
-def launch_loc(ctx, opts, ssh):
+def launch_localization(ctx, opts, ssh):
     gps = 'jetson/gps'
     filter = 'jetson/filter'
```

Comment:
When I meant full names, I mean names like jetson_filter, lcm_tools_echo, so it's clear later in your code which packages are being called exactly

Reply:
the idea of the launches is that they execute a variety of packages that are associated with a single sub-team, so I think the sub-team name would make more sense

Reply:
but these packages are referring to specific directories within mrover-workspace